src/components/search-results/search-results.tsx

```diff
@@ -1,20 +1,24 @@
 import React, { FC, useEffect } from "react";
+import CardComponent from "../card/card";
+import { SearchResult } from "../../models/request/search";
+import Grid from "@mui/joy/Grid";
 
 interface SearchResultsProps {
     search: string;
-    results: any[];
+    results: SearchResult[];
 }
 export const SearchResults: FC<SearchResultsProps> = ({ search, results }) => {
     return (
-        <div>
-            <p>
-                Searching for: <b>{search}</b>
-            </p>
-            <ul>
-                {results.map((result, i) => (
-                    <li key={i}>{result}</li>
-                ))}
-            </ul>
-        </div>
+        <Grid container spacing={4}>
+            <Grid xs={12}>
+                <h1>Search results for &quot;{search}&quot;</h1>
```

Comment:
I think it would be useful to display the number of results. Especially for edge cases where the search return nothing, right now, it is unclear that there is no results.

Reply:
I'm having some troubles to do this. Instead of displaying the total of results, is displaying the ammount of the current page (pagination)

Reply:
For this, change the Prop to `PaginatedData<SearchResult>` instead of an array. But, like the other things, it's not necessary. If you prefer to merge this, let's do that.